.github/workflows/terraform-ci.yml

```diff
@@ -0,0 +1,67 @@
+name: 'Terraform-ci'
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: sh
+
+jobs:
+  # Terraform init and validate configuration
+  terraform-init:
+    name: 'Terraform-init'
+    runs-on: ubuntu-latest
+    environment: dev
```

Comment:
i don't really understand what the environment is used for here

Reply:
Deleted
https://github.com/maddevsio/aws-eks-base/pull/29/commits/2fe5c46e524ce9fc6478306146fcc1d7b2c98a26